src/commonTest/kotlin/suite/TestSuiteRules.kt

```diff
@@ -62,7 +62,7 @@ internal class TestSuiteRules {
                     shouldFailWith<ResponseException> {
                         getRule(rule.objectID)
                     }
-                    ).response?.status?.value shouldEqual HttpStatusCode.NotFound.value
+                    ).response.status.value shouldEqual HttpStatusCode.NotFound.value
```

Comment:
I love how a lot of optionals are gone now :)

Reply:
I agree! :D